fe/fe-core/src/main/java/com/starrocks/connector/hive/ORCFormatDesc.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector.hive;
+
+import com.starrocks.thrift.TORCFormatDesc;
+
+public class ORCFormatDesc {
+
+    public static String PROPERTY_KEY = "orc.use_column_names";
```

Comment:
Do you have any reference for this name?

Reply:
refer to https://trino.io/docs/current/object-storage/file-formats.html#hive-orc-configuration